tests/integration/client_introspection/client_introspection_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package client_introspection
```

Comment:
suggestion: I think it would be tidier for this file (and the altari/graphiql query) to live under `schema`, for example `tests/integration/schema/client_introspection/`.

I think it might also be worth renaming this file to `simple_test` and moving the one-many test into a file called `one_many_test.go`, I am not sure how you envision the current names to scale as more tests get added.

Reply:
resolved